src/components/navigation/navigation.tsx

```diff
@@ -5,97 +5,79 @@ import styled from 'styled-components';
 import { IconTwitter } from '../icons/social/twitter';
 import { IconLinkedIn } from '../icons/social/linkedin';
 import { IconXing } from '../icons/social/xing';
+import { Grid, GridItem } from '../grid/grid';
 
-const NavWrapper = styled.div`
-  position: relative;
-  height: 517px;
-  overflow: hidden;
-
-  ${up('md')} {
-    height: 660px;
-  }
-`;
-
-const NavBackground = styled.div`
-  position: absolute;
-  top: 16vw;
-  right: 0;
-  left: 0;
-  height: 100%;
+const NavigationBackground = styled.div`
   background: #4d79ff;
-  transform: skewY(-9deg);
-  transform-origin: top left;
+  clip-path: polygon(0 20%, 100% 0, 100% 100%, 0% 100%);
```

Comment:
was this a design change that was asked for? because now the slant angle doesn't stay at a consistent angle, and it doesn't match the angle in the design anymore 

before:
![Screenshot 2020-07-06 at 10 14 35](https://user-images.githubusercontent.com/31735645/86571451-b65d5100-bf71-11ea-8f65-1aec9e8b4d56.png)

now:
<img width="1186" alt="Screenshot 2020-07-06 at 10 14 42" src="https://user-images.githubusercontent.com/31735645/86571458-bd845f00-bf71-11ea-8f4d-641bb94a8750.png">